ospo-book/content/en/06-chapter.md

```diff
@@ -1,7 +1,103 @@
+
+### Community Norms and Funding Strategies
+`🏛 Social-Gov-Oriented`
+
+Establishing community norms around the costs associated with SBOM delivery and compliance has been proposed as an idea. This could involve suggesting a donation model based on organization size to account for the rising costs of compliance. Additionally, the role of risk mitigation as a motivation for organizations to invest in open source was discussed, alongside the potential for government agencies to contribute indirectly to open source projects through third parties.
+
+### Policy and Funding Initiatives
+`🏛 Social-Gov-Oriented`
+
+The possibility of the EU mandating open source as a requirement for Horizon funding suggests a growing recognition of the value of open source at the policy level. There is an ongoing discussion on strategic funding models, like the German sovereign tech fund, and the potential for government roles in supporting open source.
+
+### Challenges in open source Sustainability
+`📈 Business-Oriented`
+
+The underfunding of crucial projects like Django, and the broader issue of maintainers being underpaid or volunteering, underscores the sustainability challenges within the open source ecosystem. The critical role of vulnerability management and the need for corporate contributions to open source have been highlighted as areas requiring attention and action.
+
+### Training and Education as Funding Avenues
+`🏛 Social-Gov-Oriented`
+
+The potential for integrating training about open source within government contracts is seen as a means to provide both funding for open source organizations and valuable training for employees. This represents an approach to leveraging existing procurement processes to support open source sustainability.
+
+
+## OSPOs for Container Capabilities
+
+Content has been curated from the [OSPOlogyLive Apeldoorn Roundtable Discussion](https://github.com/todogroup/ospology/discussions/438#discussioncomment-8620412) and based on the adoption of open source software (OSS) for container orchestration, particularly Kubernetes, and its implications for IT strategy in organizations.
+
+### OSPOs working with Container Hosting Infrastructure Teams
+
+Teams dedicated to this infrastructure, focusing on aspects such as image repositories and platform enablement, collaborate closely with OSPOs to support container infrastructure and capabilities. Initiatives like the Cloud Platform Engineering Team (CPET) has been highlighted for their efforts in publishing self-built operators, such as the Platform as a Service (PaaS) Operator, which automates tasks like initiating namespaces or projects on Kubernetes platforms.
+
+#### Commercial vs. Open Source for Kubernetes Deployment
```

Comment:
Same comment applies - can we be clearer about why we are profiling k8s ecosystem?